akka-typed/src/main/scala/akka/typed/internal/BehaviorImpl.scala

```diff
@@ -0,0 +1,77 @@
+/**
+ * Copyright (C) 2017 Lightbend Inc. <http://www.lightbend.com>
+ */
+package akka.typed
+package internal
+
+import akka.util.LineNumbers
+import akka.annotation.InternalApi
+import akka.typed.{ ActorContext ⇒ AC }
+import akka.typed.scaladsl.Actor
+
+/**
+ * INTERNAL API
+ */
+@InternalApi private[akka] object BehaviorImpl {
+  import Behavior._
+
+  private val _nullFun = (_: Any) ⇒ null
+  private def nullFun[T] = _nullFun.asInstanceOf[Any ⇒ T]
```

Comment:
we don't have dependency to akka-stream from here, I created ticket https://github.com/akka/akka/issues/22813